### DISCUSSION TITLE
Revert hard line wrapping

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [quarto-dev/quarto-cli#10836] [Feature Requests]
### Description

I was experimenting with 
```
editor: 
  markdown: 
    wrap: 72
```
frontmatter and set a very low value.

Next time I opened RStudio, my Quarto document was hard line wrapped to this low value.

Setting it back to a higher value did not undo the wrapping.

I ended up using pandoc to [undo the line wrapping](https://stackoverflow.com/a/70740416/1719931), but it removes the frontmatter and convert the comments as raw HTML code.

I ask for  `quarto convert` to have a `--wrap=none` option as well.

**Comment by mcanouil** [comment]
> I ask for quarto convert to have a --wrap=none option as well.

This won't work, how could Quarto know what to "revert"?

I strongly suggest to use Git or equivalent.

**Reply by raffaelemancuso** [reply]
Yes you are right
